### PR TITLE
Alpha Bug Fix II (Check, DG)

### LINKED ISSUE #155: Check Invalid Index Bug
Exception not thrown for invalid index.

### LINKED ISSUE #157: DG Bug (View Client, Use Cases)
### View Client Typo
Screenshot 2024-10-28 at 1 08 03 AM

### PR DESCRIPTION
**Pull request by xhamyo**
# Fixes #155 

### Throw exception for invalid index.
Screenshot 2024-10-28 at 12 49 36 AM

# Fixes #157 
### Fixed View Client Implementations Typo.
Screenshot 2024-10-28 at 1 10 03 AM

### Update Use Cases and Fixed Typos.

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddClientCommandTest.java
Comment (Meowloid) [inline src/test/java/seedu/address/logic/commands/AddClientCommandTest.java:1]: sorry to nitpick, but is there a reason why there's an extra tab there

**Review by Meowloid — COMMENTED** [review]
LGTM, but might wanna take a look at the comment i left

File: src/test/java/seedu/address/logic/commands/AddClientCommandTest.java
Comment (limyuquan) [reply src/test/java/seedu/address/logic/commands/AddClientCommandTest.java:1]: bc newline is 2 tabs

**Review by limyuquan — APPROVED** [review]
lgtm

File: src/test/java/seedu/address/logic/commands/AddClientCommandTest.java
Comment (xhamyo) [reply src/test/java/seedu/address/logic/commands/AddClientCommandTest.java:1]: See Java Coding Standard: https://se-education.org/guides/conventions/java/intermediate.html
`Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.`

**Pull request merged**